packages/check/match.js

```diff
@@ -379,14 +379,14 @@ var testSubtree = function (value, pattern) {
   for (var keys = allKeys(value), i = 0, length = keys.length; i < length; i++) {
     var key = keys[i];
     var subValue = value[key];
-    if (_.has(requiredPatterns, key)) {
+    if (Object.prototype.hasOwnProperty.call(requiredPatterns, key)) {
```

Comment:
I usually save a reference to this method in a `hasOwn` variable at the top of the module.